py/surveyqa/summary.py

```diff
@@ -85,6 +85,23 @@ def get_skyplot(exposures, tiles, width=500, height=250):
 
     return fig
 
+def get_median(attribute, exposures):
```

Comment:
Use a more descriptive name, to indicate that this gets the median per night and not overall median.  +1 that you wrapped this concept in a function to avoid repeated code in `get_summarytable`.